js/time.js

```diff
@@ -1,10 +1,10 @@
 export const dateString = (
     timestamp = null,
     separator = "/",
-    { year = true, month = true, day = true } = {}
+    { year = true, month = true, day = true, reverse = false } = {}

```

Comment:
I would rename to `reverseDate` so that it becomes obvious what is being reversed. For example, this could also mean reverse time (which would be very weird) or reverse the order (time appears before the date).